src/Styles/useDonationStyles.js

```diff
@@ -70,6 +88,38 @@ export default makeStyles(theme => ({
             margin: theme.spacing(15, "auto"),
         },
     },
+    GTButtonContainerStyle: {
+        marginTop: theme.spacing(8),
+        marginBottom: theme.spacing(8),
+        display: "flex",
+        justifyContent: "center",
+    },
+    flexColumn: {
+        display: "flex",
+        flexDirection: "column",
+    },
+    centerImageBox: {
+        width: "100%",
+        display: "flex",
+        justifyContent: "center",
+        overflow: "hidden",
+    },
+    centeredImage: {
+        height: 300,
+        width: 500,
+        maxHeight: { xs: 235, md: 300 },
+        maxWidth: { xs: 338, md: 500 },
+        marginLeft: "auto",
+        marginRight: "auto",
+    },
+    customHero: {
+        [theme.breakpoints.up("xs")]: {
+            minHeight: "870px",
+        },
+        [theme.breakpoints.up("md")]: {
+            minHeight: "fit-content",
+        },
+    },
     // darkBlueBackground: {
```

Comment:
I know you didn't add this code, but would you mind deleting it since it's commented out anyways?